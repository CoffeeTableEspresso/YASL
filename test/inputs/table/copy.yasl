let x = {1:'one', 2:'two', 3:'three'};
for e <- x->copy() { echo e; echo x[e]; }
