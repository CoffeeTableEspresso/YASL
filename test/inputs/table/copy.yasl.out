2
two
1
one
3
three
